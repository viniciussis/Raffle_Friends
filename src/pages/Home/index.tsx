import ParticipantsList from '../../components/ParticipantsList'
import Footer from '../../components/Footer'
import Form from '../../components/Form'
import Card from '../../components/Card'

const Home = () => {
  return (
    <Card>
      <main>
        <h1
          style={{
            color: 'orange',
            marginBottom: '2rem',
            textAlign: 'center',
            fontSize: '2rem',
          }}
        >
          Vamos come√ßar!
        </h1>
        <Form />
        <ParticipantsList />
        <Footer />
      </main>
    </Card>
  )
}

export default Home
