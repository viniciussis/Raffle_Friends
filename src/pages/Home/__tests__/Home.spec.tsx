import { BrowserRouter } from 'react-router-dom'
import { render } from '@testing-library/react'
import { RecoilRoot } from 'recoil'

import Home from '..'

describe('A home page', () => {
  it('should be rendered correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <RecoilRoot>
          <Home />
        </RecoilRoot>
      </BrowserRouter>,
    )
    expect(container).toMatchSnapshot()
  })
})
