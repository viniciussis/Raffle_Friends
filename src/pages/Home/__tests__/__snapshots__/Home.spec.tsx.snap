// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A home page should be rendered correctly 1`] = `
<div>
  <div
    class="card"
  >
    <main>
      <h1
        style="color: orange; margin-bottom: 2rem; text-align: center; font-size: 2rem;"
      >
        Vamos come√ßar!
      </h1>
      <form>
        <div
          class="formGroup"
        >
          <input
            placeholder="Insira o nome dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
            type="submit"
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul
        style="display: flex; flex-direction: column; align-items: center; gap: 0.5rem;"
      />
      <section
        class="footer"
      >
        <button
          class="footer__button"
          disabled=""
        >
          Iniciar Brincadeira!
        </button>
        <img
          alt="sacolas"
          src="/images/sacolas.png"
        />
      </section>
    </main>
  </div>
</div>
`;
